Userland/Libraries/LibWeb/SVG/SVGGraphicsElement.cpp

```diff
@@ -75,7 +75,7 @@ Gfx::AffineTransform transform_from_transform_list(ReadonlySpan<Transform> trans
 {
     Gfx::AffineTransform affine_transform;
     auto to_radians = [](float degrees) {
-        return degrees * (AK::Pi<float> / 180.0f);
+        return AK::to_radians(degrees);
     };
```

Comment:
Looks like you can get rid of this little lambda now and make its users call `AK::to_radians` directly.

Reply:
Good point